portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -30,11 +31,11 @@
   private static final String ENTITY_PROPERTY_KEY_5 = "time";
   private static final String ENTITY_PROPERTY_KEY_6 = "address";
   private static final String ENTITY_PROPERTY_KEY_7 = "description";
+  private static final double metersInAMile = 1609.34;
```

Comment:
Name should be all caps `METERS_IN_MILE`. This is so when you see it later, you immediately know that it's a constant value that won't change instead of a local variable.